docs/docsite/rst/reference_appendices/general_precedence.rst

```diff
@@ -0,0 +1,132 @@
+.. _connection_precedence_rules:
+
+Controlling Ansible connections: precedence rules
```

Comment:
i named 'general precedence' as this affects a lot more than connections

Reply:
resolved in ffec082fb09670b8a8d9b5ee88a951120972729d